ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -435,14 +435,20 @@ func (r *RayServiceReconciler) reconcileRayCluster(ctx context.Context, rayServi
 		// If no fields are set, zero downtime upgrade by default is enabled.
 		// Spec.UpgradeStrategy takes precedence over ENABLE_ZERO_DOWNTIME.
 		enableZeroDowntime := true
+		strategyMessage := ""
 		if zeroDowntimeEnvVar != "" {
 			enableZeroDowntime = strings.ToLower(zeroDowntimeEnvVar) != "false"
+			strategyMessage = fmt.Sprintf("Zero Downtime Environmental Variable is set to `%s`, zero downtime upgrade is set to `%t`", strings.ToLower(zeroDowntimeEnvVar), enableZeroDowntime)
```

Comment:
Can you include the full environment variable in this message?
```
ENABLE_ZERO_DOWNTIME environment variable is set to ...
```

Reply:
Done!